Null pointer exception after update to 0.27.0 from 0.26.1

### Description

```java
Caused by: java.lang.NullPointerException
    at io.fabric8.maven.docker.StartMojo.prepareStart (StartMojo.java:349)
    at io.fabric8.maven.docker.StartMojo.executeInternal (StartMojo.java:149)
    at io.fabric8.maven.docker.AbstractDockerMojo.execute (AbstractDockerMojo.java:225)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:134)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:51)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:309)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:194)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:107)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:955)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:290)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:194)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
```

### Info

```
* Maven version (`mvn -v`) :3.5.2
* Docker version : Docker version 18.06.1-ce, build e68fc7a
```

My configuration:

```xml
<plugin>
    <groupId>io.fabric8</groupId>
    <artifactId>docker-maven-plugin</artifactId>
    <configuration>
        <logStdout>true</logStdout>
        <logDate>default</logDate>
        <autoPull>true</autoPull>
        <removeVolumes>true</removeVolumes>
        <verbose>true</verbose>
        <images>
            <image>
                <name>xyz/database:${project.version}</name>
                <run>
                    <exposedPropertyKey>xyz-database-postgresql</exposedPropertyKey>
                    <log>
                        <enabled>true</enabled>
                        <driver>
                            <name>
                                <!-- Make sure default docker logging mode is used -->
                                json-file
                            </name>
                        </driver>
                    </log>
                    <net>bridge</net>
                    <wait>
                        <healthy>true</healthy>
                        <time>30000</time>
                        <kill>5000</kill>
                        <shutdown>5000</shutdown>
                    </wait>
                </run>
            </image>
        </images>
    </configuration>
</plugin>
```

Thank you!

--- a/src/main/java/io/fabric8/maven/docker/StartMojo.java
+++ b/src/main/java/io/fabric8/maven/docker/StartMojo.java
@@ -11,6 +11,7 @@ package io.fabric8.maven.docker;
 import java.io.IOException;
 import java.util.ArrayDeque;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
@@ -345,10 +346,10 @@ public class StartMojo extends AbstractDockerMojo {
                                                          queryService.hasImage(imageConfig.getName()));
 
             NetworkConfig config = runConfig.getNetworkingConfig();
-            RunVolumeConfiguration runVolumeConfig = runConfig.getVolumeConfiguration();
-            if(!runVolumeConfig.getBind().isEmpty()) {
+            List<String> bindMounts = extractBindMounts(runConfig.getVolumeConfiguration());
+            if(!bindMounts.isEmpty()) {
                 List<VolumeConfiguration> volumes = getVolumes();
-                runService.createVolumesAsPerVolumeBinds(hub, runVolumeConfig.getBind(), volumes);
+                runService.createVolumesAsPerVolumeBinds(hub, bindMounts, volumes);
             }
             if (autoCreateCustomNetworks && config.isCustomNetwork()) {
                 runService.createCustomNetworkIfNotExistant(config.getCustomNetwork());
@@ -357,6 +358,13 @@ public class StartMojo extends AbstractDockerMojo {
             updateAliasesSet(imageAliases, imageConfig.getAlias());
         }
         return imagesWaitingToStart;
+    }
+
+    private List<String> extractBindMounts(RunVolumeConfiguration volumeConfiguration) {
+        if (volumeConfiguration == null) {
+            return Collections.emptyList();
+        }
+        return volumeConfiguration.getBind() != null ? volumeConfiguration.getBind() : Collections.emptyList();
     }
 
     private String determinePullPolicy(RunImageConfiguration runConfig) {

